index.html

```diff
@@ -2,12 +2,189 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-
     <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/brands.min.css"
+    />
+    <title>JJ's Portfolio</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <!-- Navbar -->
+    <nav>
+      <ul>
+        <li><a href="#aboutme">About Me</a></li>
+        <li><a href="#portfolio">Portfolio</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <!-- Introduction homepage -->
+    <div class="home">
+      <div class="grid">
+        <div id="home-image">
+          <img src="images/Pang-Jin-Jia.webp" alt="" />
+        </div>
+
+        <div id="home-intro">
+          <h1>Hi there! I am JJ</h1>
+          <h1><i>A budding</i> <code>[ SOFTWARE_ENGINEER ]</code> :)</h1>
+        </div>
+      </div>
+    </div>
+
+    <!-- About me -->
+    <div id="aboutme">
+      <div class="grid">
+        <div>
+          <dl>
+            <dt><i class="fa fa-user" aria-hidden="true"></i> Profile:</dt>
+            <br />
```

Comment:
i think breaks can be a bit finnicky to manage if there are a lot of styles. better to have used a `ul` or table or something similar, then use CSS to get rid of the bullet point styling or border stylings